File: PRAK204-251087220019-TysaLuthfia.py
jar      = float(input("Jari-jari       : ")) 
ti        = float(input("Tinggi        : ")) 
vol = jar * jar * ti * 22 / 7
lu = 2 * jar * (jar + ti) * 22 / 7
kel = 2 * jar * 22 / 7
print("Hasil Volume	= {:.2f} ".format(vol))
print("Hasil Luas	= {:.2f} ".format(lu))
print("Keliling	= {:.2f} ".format(kel))

File: PRAK204-251087220019-TysaLuthfia.c
#include<stdio.h> 
int main(){
float ja, ti, vol, lu, kel;
printf("Masukkan Jari-jari	: ");
scanf("%f", &ja);
printf("Masukkan Tinggi	: ");
scanf("%f", &ti);
vol = 22 * ja * ja * ti /7;
lu = 2 * 22 * ja * (ja + ti) / 7; 
kel = 2 * 22 * ja / 7;
printf("Hasil Volume	:%.2f \n", vol);
printf("Hasil Luas	:%.2f \n", lu);
printf("Keliling	:%.2f \n", kel);
return 0;
}
